### PR TITLE
fix cronjob error

### PR DESCRIPTION
**Pull request by jeremyxu2010**

### CODE REVIEW

File: docker/files/cli/commands/cluster.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ command_remove() {
 commands_add "sync:default-hostgroup" "Synchronize the default hostgroup from backends to the cluster"
 command_sync:default-hostgroup() {
 
-    proxysql_wait_for_admin
+    proxysql_wait_for_admin_job
```
Comment (jmdejager) [inline docker/files/cli/commands/cluster.sh:75]: Shouldn't we use both? proxysql_wait_for_admin to wait for the settings to be loaded an proxysql_wait_for_admin_job te check if the cluster is up.

Comment (jeremyxu2010) [reply docker/files/cli/commands/cluster.sh:75]: `sync:default-hostgroup` is running in a cronjob pod where `PROXYSQL_SERVICE` can not be accessed by `127.0.0.1`。

Comment (jmdejager) [reply docker/files/cli/commands/cluster.sh:75]: You are right, my mistake. Thanks for your contribution!

**Comment by jeremyxu2010** [comment]
@jmdejager I use this helm chart to install proxysql cluster. Now I have some problems:

Scenario 1: A proxysql instance in the cluster crashes, although the mysql_servers and mysql_users configurations of the proxysql instance have been saved to DISK, but since the instance does not mount the persist volume, kubernetes restarts the proxysql instance, and the mysql_servers and mysql_users configurations are completely lost. This, this causes proxysql to work abnormally.

Scenario 2: A proxysql instance in the cluster crashes. After the kubernetes restarts the proxysql instance, the IP of the instance changes. Therefore, the original proxysql_servers configuration is incorrect. This also causes the proxysql to not work properly.

How to solve these problems? My goal is that the proxysql cluster can provide service to the public with high availability. A proxysql instance can self-heal after crash.

**Comment by jeremyxu2010** [comment]
@jmdejager I finally create a [new helm chart](https://github.com/jeremyxu2010/proxysql-cluster) to resolve the problems above.

**Pull request merged**